fyde-app/linux.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: Linux
+parent: Fyde App
+nav_order: 1
+---
+# Fyde App - Linux
+
+Instructions to install the Fyde App in different distributions.
+
+There are two versions available:
+
+- fyde - complete GUI version (includes fyde-tunnel)
+- fyde-tunnel - CLI version
+
+Supported versions:
+
+| Distro    | Versions      | Instructions              |
+| ---       | ---           | ---                       |
+| Arch      | -             | In progress               |
+| CentOS    | 7, 8          | [YUM](##yum-repository)   |
+| Debian    | 9, 10         | [APT](##apt-repository)   |
+| Linux     | x86_64        | In progress               |
+| Ubuntu    | 16.04, 18.04  | [APT](##apt-repository)   |
+
+## APT Repository
+
+1. Install pre-requisites
+
+    ```sh
+    sudo apt update
+    sudo apt install -y apt-transport-https wget gnupg2
+    ```
+
+1. Add repository
+
+    ```sh
+    REPO_URL="downloads.fyde.com"
+    wget -q -O - "https://$REPO_URL/fyde-public-key.asc" | sudo apt-key add -
+    sudo bash -c "cat > /etc/apt/sources.list.d/fyde.list <<EOF
+    deb https://$REPO_URL/apt stable main
+    EOF"
+    sudo apt update
+    ```
+
+1. Install
+
+    - GUI version
+
+    ```sh
+    sudo apt install fyde
+    ```
+
+    - CLI version
+
+        - Install
+
+        ```sh
+        sudo apt install fyde-tunnel
+        ```
+
+        - Enroll
+
+        ```sh
+        /opt/fyde/fyde-tunnel --enroll "<Fyde App enrollment link>"
+        sudo systemctl --system restart fyde-interface
```

Comment:
This is not needed, it is started when the package fyde-tunnel is installed.

Reply:
Got it. Removed.